models/ACC/ACC.jl

```diff
@@ -1,82 +1,112 @@
 
 using NeuralNetworkAnalysis
 
+const μ = 0.0001 # friction parameter
+const a_lead = -2.0 # acceleration control input applied to the lead vehicle
+
 @taylorize function ACC!(dx, x, p, t)
-    local u, a_lead, a_ego = 0.0001, -2., -1.
-    x_lead, v_lead, γ_lead, x_ego, v_ego, γ_ego = x
+    x_lead = x[1] # lead car position
+    v_lead = x[2] # lead car velocity
+    γ_lead = x[3] # lead car internal state
 
+    x_ego = x[4] # ego car position
+    v_ego = x[5] # ego car velocity
+    γ_ego = x[6] # ego car internal state
+    a_ego = x[7] # ego car acceleration control input
+
+    ## lead car dynamics
     dx[1] = v_lead
     dx[2] = γ_lead
-    dx[3] = 2 * (a_lead - γ_lead) - u * v_lead^2
+    dx[3] = 2 * (a_lead - γ_lead) - μ * v_lead^2
+
+    ## ego car dynamics
     dx[4] = v_ego
     dx[5] = γ_ego
-    dx[6] = 2 * (a_ego - γ_ego) - u * v_ego^2
+    dx[6] = 2 * (a_ego - γ_ego) - μ * v_ego^2
+    dx[7] = zero(a_ego)
     return dx
 end
 
-# define the initial-value problem
-X₀ = Hyperrectangle(low=[90., 32, 0, 10, 30, 0], high=[110, 32.2, 0, 11, 30.2, 0])
-
-prob = @ivp(x' = ACC!(x), dim: 6, x(0) ∈ X₀)
-
-# solve it
-##sol = solve(prob, T=0.1);
-
 # ## Specifications
 #
 # The verification objective of this system is that given a scenario where both
 # cars are driving safely, the lead car suddenly slows down with``a_{lead} = -2``.
-# We want to checkwhether there is a collision  in  the  following  5 seconds.
-# Formally, this safety specification ofthe system can be expressed as
-# ``D_{rel} = x_{lead} - x_{ego} ≥ D_{safe}``, where
-# ``D_{safe} = D_{default} + T_{gap} × v_{ego}``, and
-# ``T_{gap} = 1.4`` seconds and ``D_{default} = 10``. The initial conditions are:
-# ``x_{lead}(0) ∈ [90,110], v_{lead}(0) ∈ [32,32.2], γ_{lead}(0) = γ_{ego}(0) = 0``,
-# ``v_{ego}(0) ∈ [30, 30.2], x_{ego} ∈ [10,11]``.
+# We want to check whether there is a collision in the following 5 seconds.
+# A control period of 0.1 seconds is used.
+#
+# Formally, the safety specification can be expressed as:
+# ``D_{rel} = x_{lead} - x_{ego} ≥ D_{safe}``, where ``D_{safe} = D_{default} + T_{gap} × v_{ego}``,
+# ``T_{gap} = 1.4`` seconds and ``D_{default} = 10``.
+
+# The uncertain initial conditions are chosen to be:
+#
+# - ``x_{lead}(0) ∈ [90,110], v_{lead}(0) ∈ [32,32.2], γ_{lead}(0) = γ_{ego}(0) = 0``
+# - ``v_{ego}(0) ∈ [30, 30.2], x_{ego} ∈ [10,11]``
 
 # ## Results
 
+# The initial states according to the specification are:
+X₀ = Hyperrectangle(low=[90, 32, 0, 10, 30, 0], high=[110, 32.2, 0, 11, 30.2, 0])
+U₀ = Singleton([-1.0])
+
+# The system has 6 state variables and 1 control variable:
+vars_idx = Dict(:state_vars=>1:6, :input_vars=>[], :control_vars=>7)
+ivp = @ivp(x' = ACC!(x), dim: 7, x(0) ∈ X₀×U₀);
+
+# We will evaluate the controller which has 5 hidden layers.
+using MAT
+models_dir = normpath(@__DIR__, "..", "..", "..", "models")
+path = joinpath(models_dir, "ACC", "controller_5_20.mat")
+controller = read_nnet_mat(path, act_key="act_fcns")
```

Comment:
this directly loads the .mat file from the source repo (https://github.com/verivital/ARCH-COMP2020) and converts to a `NeuralVerification.Network` struct.

i don't know how to use `@relpath` consistently, so i would suggest to leave that simplification for later and use `normpath` as above to load the networks in the generated scripts